p2p/node.go

```diff
@@ -195,28 +195,7 @@ func (n *QriNode) GoOnline() (err error) {
 	n.Online = true
 	go n.echoMessages()
 
-	return n.startOnlineServices()
-}
-
-// startOnlineServices bootstraps the node to qri & IPFS networks
-// and begins NAT discovery
-func (n *QriNode) startOnlineServices() error {
-	if !n.Online {
-		return nil
-	}
-
-	bsPeers := make(chan pstore.PeerInfo, len(n.cfg.BootstrapAddrs))
-
-	go func() {
-		// block until we have at least one successful bootstrap connection
-		<-bsPeers
-
-		if err := n.AnnounceConnected(n.Context()); err != nil {
-			log.Infof("error announcing connected: %s", err.Error())
-		}
-	}()
-
-	return n.StartDiscovery(bsPeers)
+	return nil
```

Comment:
This is the only removal I don't totally understand. I know we removed the `BootstrapAddrs`, but how do we now bootstrap the node? Is this something that ipfs takes care of in its newest versions? Or is this a change that happened in a pr that went by that I didn't fully read?
tagging @b5 if he has insight.

Reply:
Yeah, we definitely need to keep the `startOnlineServices` function.